src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -6,8 +6,8 @@
     "System.Net.Http": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-23921",
-    "System.ServiceModel.Duplex": "4.0.1-rc3-23921",
-    "System.ServiceModel.Http": "4.0.11-rc3-23921",
+    "System.ServiceModel.Duplex": "4.1.0-rc3-23921",
```

Comment:
After updating all the test projects to System.Linq.Expressions 4.0.11-rc3-23921 I no longer have any package restoration failures but now I am back to the System.IO.FileNotFoundException for System.Security.SecureString version 4.0.0

So looks like somehow we are still getting an old version of System.Net.Http.WinHttpHandler

Continuing to investigate.